src/shared/dropdown-wrapper/dropdown-wrapper.styles.tsx

```diff
@@ -189,10 +189,17 @@ export const Divider = styled.div`
     margin: 0 0.5rem;
 `;
 
-export const LabelContainer = styled.div`
+export const LabelContainer = styled.div<{ disabled?: boolean }>`
```

Comment:
declare the prop in an interface under the `STYLE INTERFACE` section instead of inlining them per the repo conventions